src/commands/google.ts

```diff
@@ -0,0 +1,86 @@
+import {CommandType} from "../types";
+import {logger} from "@/config";
+
+import {
+  CacheType,
+  SlashCommandBuilder,
+  ChatInputCommandInteraction,
+  EmbedBuilder,
+  Colors,
+} from "discord.js";
+
+const googleModule: CommandType = {
+  data: new SlashCommandBuilder()
+    .setName("google")
+    .setDescription("Google something and return the top 10 results.")
+    .addStringOption((option) =>
+      option
+        .setName("query")
+        .setDescription("What you want to Google.")
+        .setRequired(true)
+    )
+    .addUserOption((option) =>
+      option
+        .setName("to")
+        .setDescription("The user you want to direct this to.")
+    ),
+  execute: async (interaction: ChatInputCommandInteraction<CacheType>) => {
+    const query = interaction.options.getString("query");
+    const toWhom = interaction.options.getUser("to");
+    const embed = new EmbedBuilder()
+      .setColor(Colors.Blue)
+      .setAuthor({
+        name: `Requested by ${interaction.user.displayName}`,
+        iconURL: `${interaction.user?.avatarURL()}`,
+      })
+      .setTimestamp();
+
+    const search_key = "insert custom search api token";
```

Comment:
Is there a reason these aren't provided by the `Config`?

Reply:
> Is there a reason these aren't provided by the `Config`?

Didn't want to add more config variables, but if it is not an issue then I'll add them as environment variables like I did in my last pull request.

Reply:
Its far more convenient than requiring a code change. Stuff like this should always be part of the config (or environment variables, but config is generally nicer).